### ISSUE TITLE
Desktop language list is becoming too long

### ISSUE THREAD

**Issue by danieldegroot2** [openstreetmap-polska/openaedmap-frontend#108]
The desktop language list is becoming too long. In order for the user to be able to see all languages, the list should be limited to fewer entries and have a scroll bar to allow the user to see all the languages.
If the user zooms in until it switches to the mobile layout (or has a small screen), all languages should remain available as well.

[MapComplete](https://mapcomplete.osm.be/) has such a menu too.

Related: #63

**Comment by ttomasz** [comment]
Tips if someone wants to implement it.

CSS that could make the list scrollable is:
```css
.navbar-dropdown {
    max-height: 30em;
    overflow: auto;
}
```
steps to implement:
1. create languageSwitcher.css
2. add code from above
3. add `import './languageSwitcher.css';` to languageSwitcher.js

But it would require some testing to see if that would be better or other changes would need to be done as well. Default scroll bar is not very visible on green background for example. Additionally expecially on mobile there is a problem that Navbar.Item elements are block which does not allow overlap so maybe something (hide logo/openaedmap text when language menu open?) would also need to be done here to have nice mobile UI.

Alternatively we could use library like `react-select` which i plan to use for list of countries to make the list searchable and scrollable.

**Comment by LaoshuBaby** [comment]
I agree, because currently language list already can't display correctly when in small window size:

![图片](https://user-images.githubusercontent.com/42690037/213841654-bf031660-cdf6-4eb6-b7a9-4084912446e2.png)
![图片](https://user-images.githubusercontent.com/42690037/213841660-f7d71c33-1618-48dd-90e8-222bea810aec.png)

**Comment by starsep** [comment]
Implementation of mobile language picker using select is available at https://dev.openaedmap.org/
Resolved in b3c485d16059dfbfd449fe3418d13c1f21f1b72b

| Before | After |
| ----------|----------|
| ![image](https://github.com/openstreetmap-polska/openaedmap-frontend/assets/2798728/267119db-9888-4c89-bac5-87d68dbad2c5) | ![image](https://github.com/openstreetmap-polska/openaedmap-frontend/assets/2798728/f0db9956-b9fd-461f-b82f-9c35ad21de75) |
| ![image](https://github.com/openstreetmap-polska/openaedmap-frontend/assets/2798728/05c1fc8f-2022-4d87-8004-cd21ee0647e2) | ![image](https://github.com/openstreetmap-polska/openaedmap-frontend/assets/2798728/7f4cac67-05e8-48e4-8b60-eb2591f704ca) |
